api/types/types.go

```diff
@@ -84,7 +83,7 @@ type DiskUsageOptions struct {
 	Types []DiskUsageObject
 }
 
-// DiskUsage contains response of Engine API:
+// DiskUsage contains response of Engine API deprecated in API 1.49.
 // GET "/system/df"
```

Comment:
The deprecation notice needs to be put in a separate paragraph to be correctly interpreted: https://go.dev/wiki/Deprecated

```suggestion
// DiskUsage contains response of Engine API:
// GET "/system/df"
//
// Deprecated: deprecated in API 1.49, use [system.DiskUsage] instead.
```

Reply:
I removed this deprecation note for now. The `system.DiskUsage` struct still exists but since `/system/df` is still returning it it became a bit too complicated to deprecate it since the struct still needed to be used. That would have resulted in a `nolint` in a lot of places that didn't make sense.

We can likely fix the immediate issue and do the deprecation in something different. Alternatively, we can maybe just create a new endpoint for this instead that uses the new type so we aren't causing a big inconvenience with other clients.